### PR TITLE
Bookstore styling

### PR DESCRIPTION
**Pull request by Rachelwebdev**
In this Milestone I;

- Added JSX elements to the bookstore

- Matched final design with Zeplin's design 

- Used the CSS rules detailed in Zeplin to style the website.

- Fixed linter errors

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -14,17 +14,56 @@ function Book({ title, author, id }) {
...
+        <p>{title}</p>
+        <p>{author}</p>
+        <button
+          data-id={id}
+          type="button"
+          onClick={removeHandler}
+        >
+          remove
+        </button>
+      </li> */}
```
Comment (pinkmoon25) [inline src/components/Book.js]: - [ ] Great work with styling the app, however I cannot yet approve the project as you have commented code in your codebase following the JS best practices it is recommended to not commit old pieces of code as inline comments. They will make the project look messy. please fix this

**Review by pinkmoon25 — CHANGES_REQUESTED** [review]
Hi @Rachelwebdev , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
linter checks :heavy_check_mark: 
gitflow :heavy_check_mark: 
Styling :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

**[OPTIONAL] Please fix the Live demo link on your Readme as currently it is broken and sowing a 404 error** :pray: 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @Rachelwebdev , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] No linter errors :white_check_mark: 
- [x] Gitflow is correctly used :white_check_mark: 
- [x]  Descriptive pull request title and summary :white_check_mark:  
- [x] Implemented all the changes from the previous reviewer :white_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Rachelwebdev** [comment]
Thanks for the approval 👍

**Pull request merged**